api/src/main/java/jakarta/servlet/http/Cookie.java

```diff
@@ -441,9 +441,19 @@ public boolean isHttpOnly() {
      * except for <code>version</code>. E.g. when <code>cookie.setAttribute("domain", domain)</code> is invoked, then
      * <code>cookie.getDomain()</code> should return exactly that value, and vice versa.
      *
+     * <p>
+     * Attributes with a value of the empty string will appear as {@code "attribute-name="} in any {@code Set-Cookie} header
```

Comment:
```suggestion
     * Attributes with a value of the empty string will appear as {@code "attribute-name"} (a cookie without a value or equals sign) in any {@code Set-Cookie} header
```

Reply:
Applied with the minor change of "...cookie **attribute** without a value..."